website/content/en/docs/main/getting-started/users.md

```diff
@@ -4,12 +4,12 @@ description: Installing and configuring a basic ClusterLink deployment
 weight: 22
 ---
 
-This guide will give you a quick start on installing and setting up the ClusterLink on a Kubernetes cluster.
+This guide will give you a quick start on installing and setting up ClusterLink on a Kubernetes cluster.
 
 ## Prerequisites
 
 Before you start, you must have access to a Kubernetes cluster.
-For example, you can set up a local environment using [kind][].
+For example, you can set up a local environment using [kind](https://kind.sigs.k8s.io/docs/user/quick-start/).
```

Comment:
we intentionally removed all link targets from the code to the bottom of markdown files. It makes them easier to locate and change. It also makes link changes easier to spot (vs content changes)